src/anyio/from_thread.py

```diff
@@ -454,7 +454,9 @@ def __exit__(
 
 @contextmanager
 def start_blocking_portal(
-    backend: str = "asyncio", backend_options: dict[str, Any] | None = None
+    backend: str = "asyncio",
+    backend_options: dict[str, Any] | None = None,
+    name: str | None = None,
```

Comment:
I would prefer this to be a keyword-only argument.
```suggestion
    *,
    name: str | None = None,
```